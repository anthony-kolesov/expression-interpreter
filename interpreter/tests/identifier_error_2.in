# Check error for case where identifier is declared sometime later.
out g1
var g1 = 2
out g1
print "\n"
