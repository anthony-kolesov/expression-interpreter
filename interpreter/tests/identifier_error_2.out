ERROR:2:Unknown identifier: g1
